### DISCUSSION TITLE
I want all Stored Procedures

### DISCUSSION THREAD

**Discussion by arjunbharara** [Mimetis/Dotmim.Sync#1269] [Q&A]
When I perform a standard provisioning, it creates some of the selected stored procedures, but it doesn’t generate the SelectRow and SelectChanges stored procedures, which are available in the DbStoredProcedureType enum.

Is there a way to ensure these stored procedures are generated for all tables without having to run a loop? I attempted to use the OnProvisioningTable interceptor, but it didn’t work as expected.

**Comment by Mimetis** [comment]
Depending on the provider you are using, stored procedures could be in memory (with a single statement)
For `SqlSyncProvider`:
- `SelectRow` is an in memory statement
- `SelectChanges` is a stored procedure (usually name "`table_scope_changes`" )
